src/utils.rs

```diff
@@ -405,24 +405,24 @@ pub async fn should_be_installed(
         return false;
     }
 
+    let jurisdiction_preferences = hosting_preferences.jurisdiction_prefs.unwrap();
```

Comment:
do you want the whole script to panic and stop if `jurisdiction_prefs` is `None`?

Reply:
No I do not.